Content.Server/GameObjects/EntitySystems/MoverSystem.cs

```diff
@@ -153,6 +153,12 @@ private void UpdateKinematics(ITransformComponent transform, IMoverComponent mov
                 if (mover.StepSoundDistance > distanceNeeded)
                 {
                     mover.StepSoundDistance = 0;
+
+                    if (mover.Owner.Prototype.ID == "MobObserver" || mover.Owner.Prototype.ID == "AdminObserver")
```

Comment:
This should absolutely be done by checking for the existence of a component.

Make a component that makes something make footstep sounds and add it to humans.

Reply:
Added `FootstepSoundComponent`. It's just an empty component that's checked for in `MoverSystem.cs` instead of checking the prototype names. Figured I'd start with that simple approach, but if you wanted it to do more lemme know.

Reply:
Nah, that's perfect.